hpat/distributed_analysis.py

```diff
@@ -68,6 +68,9 @@ def _init_run(self):
         self.second_pass = False
         self.in_parallel_parfor = -1
 
+    def run_pass(self, state):
```

Comment:
remove and call `run` directly

Reply:
I think run_pass(self, state) is interface function. It is better to inline run() here.

Reply:
renamed